src/main/java/org/csanchez/jenkins/plugins/kubernetes/ContainerTemplate.java

```diff
@@ -350,6 +354,8 @@ public FormValidation doCheckImage(@QueryParameter String value) {
         public String getWorkingDir() {
             return DEFAULT_WORKING_DIR;
         }
+        public final static String defaultCommand = "sleep";
+        public final static String defaultArgs = "9999999";
```

Comment:
```suggestion
        public final static String DEFAULT_COMMAND = "sleep";
        public final static String DEFAULT_ARGS = "9999999";
```
Customary to use all caps for `static final` vars (constants). Not sure if Jelly has different rules..

Reply:
I'll test it to see if jelly is doing some sort of transformation, and change it if not.

Reply:
done.

Reply:
Jelly does not have different rules.